application/dash_application/views/SARS-CoV-2.py

```diff
@@ -198,7 +200,7 @@ def generate_average_coverage_scatter(current_data, graph_params):
         graph_params["colour_by"],
         graph_params["shape_by"],
         graph_params["shownames_val"],
-        [(avg_coverage_cutoff_label, avg_coverage_cutoff_value)] # Should be unchanging
+        [(avg_coverage_cutoff_label, graph_params["avg_coverage_cutoff"])] # Should be unchanging
```

Comment:
I think this comment and the one below is out of date.

Reply:
Fixed!